fili-core/src/main/java/com/yahoo/bard/webservice/web/DataApiRequest.java

```diff
@@ -553,7 +559,7 @@ private static void validateTimeAlignment(
             return dimension.getDimensionFields();
         } else if (showFields.size() == 1 && showFields.contains(DimensionFieldSpecifierKeywords.NONE.toString())) {
             // Show no fields
-            return Collections.emptySet();
+            return new LinkedHashSet<>();
```

Comment:
What do you mean `singleton of this`? There are no elements, it's an empty set. Do you mean `Collections.emptySet`? Can't do that because, the `Collections.emptySet()` is of type `Set` not `LinkedHashSet`, which is the new return type for this method.